page_creation.rst

```diff
@@ -209,11 +209,11 @@ Bundles are registered in your ``app/AppKernel.php`` file (a rare PHP file in th
     {
         public function registerBundles()
         {
-            $bundles = array(
+            $bundles = [
```

Comment:
I would say no. If we change the line below this or move the example in some way, we'll always get merge conflicts.